firebase/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+''' Builds and exports Firebase dependencies.
+To reference these dependencies, add '//firebase:crashlytics_lib' and '//firebase:crashlytics_deps'
+to your build rule's dependency list.
+'''
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@tools_android//tools/crashlytics:defs.bzl", "crashlytics_android_library")
+load("@tools_android//tools/googleservices:defs.bzl", "google_services_xml")
+
+package(default_visibility = ["//visibility:public"])
+
+GOOGLE_SERVICES_RESOURCES = google_services_xml(
+    package_name = "org.oppia.app",
+    google_services_json = "google-services.json",
+)
+
+crashlytics_android_library(
```

Comment:
I missed this in your earlier PR, but prefer omitting ‘lib’ from library names. It’s redundant, and it’s conventional not to include it.

Reply:
Okay!